x-pack/plugins/enterprise_search/public/applications/enterprise_search/cypress.json

```diff
@@ -0,0 +1,21 @@
+{
+  "supportFile": false,
+  "pluginsFile": false,
+  "retries": {
+    "runMode": 2
+  },
```

Comment:
Super excited for us getting retries for free with Cypress 6.8! (we were on like 4.7 in ent-search I think)